### PR TITLE
fix(Form.Control): fix `acceptor` prop can not accept `Toggle`

### PR DESCRIPTION
**Pull request by simonguo**
fix: https://github.com/rsuite/rsuite/issues/3036#issuecomment-1408365439

### CODE REVIEW

File: src/FormControl/test/FormControlSpec.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Form, { FormInstance } from '../../Form';
 import FormControl from '../FormControl';
 import FormGroup from '../../FormGroup';
 import Schema from '../../Schema';
+import Toggle from '../..//Toggle';
```
Comment (SevenOutman) [inline src/FormControl/test/FormControlSpec.tsx:10]: Incorrect import path

**Review by SevenOutman — CHANGES_REQUESTED** [review]

File: src/FormControl/test/FormControlSpec.tsx (hunk 1 above)
Comment (simonguo) [inline src/FormControl/test/FormControlSpec.tsx:10]:
```suggestion
import Toggle from '../../Toggle';
```

**Review by SevenOutman — APPROVED** [review]

**Comment by ab-pm** [comment]
This breaks code that passes a custom value to the `<Toggle>` via the `checked` property:
```js
<Form.Control
  name="…"
  accepter={Toggle}
  checked={myValue}
/>
```
Previously, the form control would forward the `checked` value to the `Toggle`. Now, it does overwrite the `...rest` with the `...accepterProps` containing a `checked` value from the form context. One can workaround this by passing it as `value`, like
```js
<Form.Control
  name="…"
  accepter={Toggle}
  value={myValue}
/>
```
but that's weird to pass a `value` prop to affect the `checked` prop.

I would suggest you either
* change the order of spreads to `{...accepterProps} {...rest}` or
* you already take the `value` and `defaultValue` not directly from `props` but rather use the custom keys, i.e. `props[valueKey]` and `props[defaultValueKey]`.

**Comment by SevenOutman** [comment]
> This breaks code that passes a custom value to the `<Toggle>` via the `checked` property:
> 
> ```js
> <Form.Control
>   name="…"
>   accepter={Toggle}
>   checked={myValue}
> />
> ```

Hi @ab-pm, as Form.Control now controls the `checked` value of Toggle, you may want to pass `myValue` to Toggle via `formValue` (which expectedly didn't work before).

**Comment by ab-pm** [comment]
Yes, it's nice that this works now. But sometimes there are cases where you want to override the value taken from the `formValue` per Form.Control, which usually works by passing it directly to `<Form.Control value={myInputText} />`. This approach is no longer working for `Toggle` accepters, which is *not intuitive*.
(An alternative solution for #3036 would have been to align `<Toggle>` with the other input components by making it take a `value` instead of a `checked` prop, but I understand this would have been a larger breaking change)
Sure I can change all my code to use `value` instead of `checked`, or in some places I might be able to adjust the `formValue` objects, but I'd prefer not having to do either of those things.

**Comment by SevenOutman** [comment]
> But sometimes there are cases where you want to override the value taken from the `formValue` per Form.Control, which usually works by passing it directly to `<Form.Control value={myInputText} />`

This use case makes sense to me. I'm going to fix it.

**Pull request merged**